SPECS/rpm/rpm.spec

```diff
@@ -266,6 +266,9 @@ popd
 %{python3_sitelib}/*
 
 %changelog
+* Fri Mar 22 2024 Sam Meluch <sammeluch@microsoft.com> - 4.18.2-1
+- Upgrade rpm to version 4.18.2
```

Comment:
Looks good but don't forget to update `cgmanifest.json` with the new URL.

Reply:
good catch, added the version bump to `cgmanifest.json`